packages/browser-wallet/src/popup/shared/Form/Password/Password.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { InputHTMLAttributes, useMemo, useState } from 'react';
+import { passwordStrength } from 'check-password-strength';
+import { Normalize, useTranslation } from 'react-i18next';
+
+import HiddenIcon from '@shared/assets/svg/hidden.svg';
+import VisibleIcon from '@shared/assets/svg/visible.svg';
+import { Input } from '../Input';
+import { makeControlled } from '../common/utils';
+import { CommonFieldProps, RequiredControlledFieldProps } from '../common/types';
+import en from '../../i18n/en';
+
+const strengthTexts: Normalize<typeof en>[] = [
+    'form.password.tooWeak',
+    'form.password.weak',
+    'form.password.medium',
+    'form.password.strong',
+];
+
+type Props = Pick<InputHTMLAttributes<HTMLInputElement>, 'className'> &
+    RequiredControlledFieldProps &
+    CommonFieldProps & {
```

Comment:
This component should not allow for `note` as an input as it generates its own `note` field from the password strength.